src/protocols/vnc/user.c

```diff
@@ -112,6 +112,8 @@ int guac_vnc_user_leave_handler(guac_user* user) {
 
     guac_vnc_client* vnc_client = (guac_vnc_client*) user->client->data;
 
+    if(!vnc_client || !user || !vnc_client->display || !vnc_client->display->cursor) return 1;
```

Comment:
To quote @mike-jumper: if is not a function.  (Should be a space between the "if" and the opening parenthesis.)

:-)

Also, to be consistent with style used throughout the rest of the code, the return one should be on its own line and indented.